pennylane/qchem/openfermion_obs.py

```diff
@@ -105,6 +103,14 @@ def observable(fermion_ops, init_term=0, mapping="jordan_wigner", wires=None):
     + (0.075) [Z2]
     + (-0.075) [Z0 Z2]
     """
+    try:
+        # pylint: disable=import-outside-toplevel, unused-import
+        import openfermion, openfermionpyscf
+    except ImportError as Error:
+        raise ImportError(
+            "This feature requires openfermion, openfermionpyscf. They can be"
+            " installed with `pip install openfermion openfermionpyscf`."
+        ) from Error
```

Comment:
To avoid the duplication, I would recommend factoring this out to a function?

```python
def import_of():
    try:
        # pylint: disable=import-outside-toplevel, unused-import
        import openfermion, openfermionpyscf
    except ImportError as Error:
        raise ImportError(
            "This feature requires openfermion, openfermionpyscf. They can be"
            " installed with `pip install openfermion openfermionpyscf`."
        ) from Error

    return openfermion, openfermionpyscf

...

openfermion, openfermionpyscf = import_of()
```

(not sure if you would need to return `openfermion` and `openfermionpyscf` from the function or not! You may need to).

Reply:
Done.